modules/images-update-global-pull-secret.adoc

```diff
@@ -9,7 +9,7 @@
 [id="images-update-global-pull-secret_{context}"]
 = Updating the global cluster pull secret
 
-You can update the global pull secret for your cluster.
+You can update the global pull secret for your cluster by either replacing the current pull secret or adding a new one in your pull secret file.
```

Comment:
@snarayan-redhat, but is this actually created in a file? I think the pull secret lives in OCP itself, not locally in a file. Because `oc get secret/pull-secret` is run, I'd say it's definitely upstream and not local. So I'd say

> by either replacing the current pull secret or appending a new pull secret. 

Or something similar.

Reply:
@jboxman Thanks Jason. It does imply an incorrect message here. The .json file will append the pull secret with additional credentials. I will make the necessary wording change.